relayer/pkg/chainlink/ocr2/medianreport/report.go

```diff
@@ -48,6 +49,7 @@ func (c ReportCodec) BuildReport(oo []median.ParsedAttributedObservation) (types
 		return oo[i].JuelsPerFeeCoin.Cmp(oo[j].JuelsPerFeeCoin) < 0
 	})
 	juelsPerFeeCoin := oo[num/2].JuelsPerFeeCoin
+	juelsPerFeeCoin = starknet.BigIntToFelt(juelsPerFeeCoin) // converts negative bigInts to corresponding felt in bigInt form
```

Comment:
We probably don't need this since this number should never be negative

Reply:
i think this might still be worth handling...I'm just thinking defensively where we always want the value to be updated onchain even though the payment might be incorrect

scenario 1 (do not convert negative val): if enough operators submit negative numbers, the median will be negative. this would cause the transaction to fail because a negative big.Int doesn't fit on the starkcurve

scenario 2 (return an error if negative): again if enough operators submit negative values, the negative median would cause an error for a valid report, and reports would stop being broadcast